lib/rules/web/ocm_console/cluster_list_page.xyaml

```diff
@@ -116,7 +119,7 @@ close_filter_dropdown:
 expand_actions_behind_create_button:
     element:
         selector:
-            xpath: //button[text()='Create Cluster']/../../../div[@id='cluster-list-toolbar']//button[@aria-label='Actions']
+            xpath: //button[text()='Create cluster']/../../../div[@id='cluster-list-toolbar']//button[@aria-label='Actions']
```

Comment:
This rule is as same as the above `//button[text()='Create cluster']/../..//button[@aria-label='Actions']`,

Reply:
And this element part should be the action `click_actions_behind_create_button:` instead of rewrite it again.